src/il/IL_helpers.ml

```diff
@@ -156,6 +157,19 @@ module LvalOrdered = struct
             (fun offset1 offset2 ->
               match (offset1.o, offset2.o) with
               | Dot a, Dot b -> compare_name a b
+              | Index { e = Operator ((G.Mult, _), []); _ }, Index _
+              | Index _, Index { e = Operator ((G.Mult, _), []); _ } ->
+                  (* This stands for an arbitrary index, like '[*]',
```

Comment:
What context can this syntax originate from? The pattern language, python, somewhere else?

Reply:
There is a `see 'Pro_taint_lval_env.lval_index_any'`, if you go see that definition and how it's used in that module, does it clarify it?

Reply:
I expanded it a bit more, and I'll some more docs in `Pro_taint_lval_env`, let me now if that's not clear enough and I can improve it!